jetty-cdi/src/main/config/etc/cdi/jetty-web-cdi.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+
+<Configure class="org.eclipse.jetty.webapp.WebAppContext" id="cdi-context">
+  <Call name="setAttribute">
+    <Arg>org.eclipse.jetty.cdi</Arg>
+    <Arg>DecoratingListener</Arg>
```

Comment:
What's the purpose of this attribute?

Reply:
This attribute is used to signal to the CDI implementation that it can use the DecoratingListener.  I guess I should comment that:)
See https://github.com/weld/core/pull/1926/files#diff-1a2ecb3dc216efb11456f5a48efd549bR44

Reply:
The other option I thought of was some optional call back on the Decorator that would inform it that it had been registered.... but that comes after the attempt and is no good for the `touch` implementation needed by weld